source/gameshared/q_comref.h

```diff
@@ -207,8 +208,12 @@ typedef struct
 #define CS_WORLDMODEL		30
 #define	CS_MAPCHECKSUM		31		// for catching cheater maps
 
+#define CS_DASHJUMP_COOLDOWN 32
```

Comment:
uhh sorry for being a little nitpicky but there should probably be a single CS_MOVEMENT or something holding all of these, you can pack them all into one string the same way that CS_SERVERINFO does